plumbing/revlist/revlist.go

```diff
@@ -1,5 +1,5 @@
-// Package revlist implementation to walk the objects referenced by a commit
-// history. Roughly equivalent to git-rev-list command.
+// Package revlist is a implementation to walk the objects referenced by a
```

Comment:
s/ a / an /

The sentence still doesn't make sense, revlist is a package, not an implementation (an implementation of what?).

Consider this other sentence instead: `Package revlist provides support to access the ancestors of commits, in a similar way as the git-rev-list command`.